app/boards/shields/gherkin/gherkin.overlay

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix-transform.h>
+
+/ {
+    chosen {
+        zmk,kscan = &kscan0;
+        zmk,matrix_transform = &default_transform;
+    };
+
+    default_transform: keymap_transform_0 {
+        compatible = "zmk,matrix-transform";
+        columns = <6>;
+        rows = <5>;
+        map = <
+    RC(0,0) RC(0,1) RC(0,2) RC(0,3) RC(0,4) RC(0,5) 
+    RC(1,0) RC(1,1) RC(1,2) RC(1,3) RC(1,4) RC(1,5) 
+    RC(2,0) RC(2,1) RC(2,2) RC(2,3) RC(2,4)  RC(2,5) 
+    RC(3,0) RC(3,1) RC(3,2) RC(3,3) RC(3,4) RC(3,5) 
+    RC(4,0) RC(4,1) RC(4,2)  RC(4,3) RC(4,4) RC(4,5)
```

Comment:
If we do keep this transform, it should be in the shape of the Gherkin (3x10?) not 5x6.